src/annotator/components/toolbar.js

```diff
@@ -100,51 +103,101 @@ export default function Toolbar({
-            label="Show highlights"
-            icon={showHighlights ? 'show' : 'hide'}
-            selected={showHighlights}
-            onClick={toggleHighlights}
+            className="annotator-toolbar__sidebar-close"
+            label="Close annotation sidebar"
+            icon="cancel"
+            onClick={closeSidebar}
           />
+        )}
+        {!useMinimalControls && (
           <ToolbarButton
-            label={
-              newAnnotationType === 'note' ? 'New page note' : 'New annotation'
-            }
-            icon={newAnnotationType === 'note' ? 'note' : 'annotate'}
-            onClick={createAnnotation}
+            className="annotator-toolbar__sidebar-toggle"
+            buttonRef={toggleSidebarRef}
+            label="Annotation sidebar"
+            icon={isSidebarOpen ? 'caret-right' : 'caret-left'}
+            expanded={isSidebarOpen}
+            onClick={toggleSidebar}
           />
+        )}
+        {!useMinimalControls && (
+          <div className="annotator-toolbar-buttonbar">
+            <ToolbarButton
+              label="Show highlights"
+              icon={showHighlights ? 'show' : 'hide'}
+              selected={showHighlights}
+              onClick={toggleHighlights}
+            />
+            <ToolbarButton
+              label={
+                newAnnotationType === 'note'
+                  ? 'New page note'
+                  : 'New annotation'
+              }
+              icon={newAnnotationType === 'note' ? 'note' : 'annotate'}
+              onClick={createAnnotation}
+            />
+            <ToolbarButton
+              label="New Doodle"
+              icon="doodle"
+              onClick={drawingToolbarToggle}
+            />
+          </div>
+        )}
+      </div>
+    );
+  } else {
+    return (
+      <div>
+        {useMinimalControls && isSidebarOpen && (
           <ToolbarButton
-            label={'New Doodle'}
-            icon={'doodle'}
-            onClick={toggleDoodleability}
+            className="annotator-toolbar__sidebar-close"
+            label="Close annotation sidebar"
+            icon="cancel"
+            onClick={closeSidebar}
           />
-        </div>
-      )}
-    </div>
-  );
+        )}
+        {!useMinimalControls && (
+          <ToolbarButton
+            className="annotator-toolbar__sidebar-toggle"
+            buttonRef={toggleSidebarRef}
+            label="Annotation sidebar"
+            icon={isSidebarOpen ? 'caret-right' : 'caret-left'}
+            expanded={isSidebarOpen}
+            onClick={toggleSidebar}
+          />
+        )}
+        {!useMinimalControls && (
+          <div className="annotator-toolbar-buttonbar">
+            <ToolbarButton
+              label="Stop doodle"
+              icon="close"
+              onClick={drawingToolbarToggle}
+            />
+            <ToolbarButton
+              label="Pen"
+              icon="pen"
+              onClick={toggleDoodleability}
```

Comment:
this should just set the tool to pen now, not toggle the doodleability, but I can add that when I add the eraser tool.